cdc/puller/frontier/frontier.go

```diff
@@ -68,7 +68,6 @@ func (s *spanFrontier) Frontier() uint64 {
 
 // Forward advances the timestamp for a span.
 func (s *spanFrontier) Forward(span regionspan.ComparableSpan, ts uint64) {
-	span = span.Hack()
```

Comment:
Is it possible to remove `span.Hack()` at https://github.com/pingcap/tiflow/pull/7203/files#diff-01b25d9f5901400302bd5f2b44a7f5e754b0037fd5fcec92d2a7acf066d25acaL39 too? It is little bit confused.